tgui/packages/tgui/interfaces/SheetCrafting.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @file
+ * @copyright 2022
+ * @author Garash (https://github.com/Garash2k)
+ * @license ISC
+ */
+
+import { useBackend } from "../backend";
+import { Button, Section, Image } from "../components";
+import { Window } from '../layouts';
+
+export const SheetCrafting = (_props, context) => {
+  const { data } = useBackend(context);
+  const { availableAmount, labeledAvailableAmount, itemList } = data;
+
+  return (
+    <Window
+      width={360}
+      height={690}>
+      <Window.Content>
+        <Section title={"Recipes"}>
```

Comment:
Can you divide these 2 sections with a `Stack` so you can make the first one scrollable and the second one is always visible? 
![image](https://user-images.githubusercontent.com/14966486/181937163-a4ac82ff-b990-4539-bb02-c1ff31888a7b.png)